lib/ukrand/chacha.c

```diff
@@ -109,15 +83,15 @@ static inline void _uk_iv_setup(struct uk_swrand *r, __u32 iv[2])
 }
 
 static inline __u32 _infvec_val(unsigned int c, const __u32 v[],
```

Comment:
If you happen to have time, you could also add an additional commit adding `const` where needed to types as well as converting them to their non-libc variant. E.g. `ssize_t` to our `__ssz` (defined in `essentials.h`).